web/client/components/map/openlayers/Map.jsx

```diff
@@ -156,6 +162,11 @@ class OpenlayersMap extends React.Component {
         this.map.enableEventListener = (event) => {
             delete this.map.disabledListeners[event];
         };
+        this.map.getViewport().addEventListener('mouseout', () => {
+            if (this.props.mousePosition) {
```

Comment:
You can trigger the mouseout anyway, without this guard. The problems of triggering events is only with very frequent events, like pointermove or similar. Here this is not needed.